linux/packaging/appimage/make_config.yaml

```diff
@@ -1,16 +0,0 @@
-display_name: Spotube
-icon: assets/spotube-logo.png
-
-keywords:
-  - Music
-  - Spotify
-  - Media
-  - Streaming
-  - YouTube
-
-generic_name: Music Streaming Application
-categories:
-  - Music
-
-supported_mime_type:
-  - x-scheme-handler/spotify
```

Comment:
Again, why the necessity to remove this file? It's important to generate a `.AppImage ` for linux platform